DataProcessing/datax-utility/pom.xml

```diff
@@ -50,11 +50,11 @@ SOFTWARE
     </scm>
 
     <groupId>com.microsoft.datax</groupId>
-    <artifactId>datax-utility_2.3_2.11</artifactId>
-    <version>1.1.0</version>
+    <artifactId>datax-utility_2.4_2.11</artifactId>
```

Comment:
did we rev our component versions, and are we avoiding snapshots in RC?

Reply:
yes we did. and yes we don't have snapshots in RC.